tests/test_fem.py

```diff
@@ -199,23 +199,32 @@ def test_maxvolume(fem_material, show_viewer, box_obj_path):
 @pytest.fixture(scope="session")
 def fem_material_linear():
     """Fixture for common FEM linear material properties"""
-    return gs.materials.FEM.Elastic()
+    return gs.materials.FEM.Elastic(friction_mu=0.5)
 
 
 @pytest.mark.parametrize("backend", [gs.cpu])
-def test_multiple_fem_entities_implicit(fem_material_linear, show_viewer):
+def test_sphere_fall_implicit(fem_material_linear, show_viewer):
```

Comment:
I have the feeling the old unit test is still relevant? I think it would be better to keep them both

Reply:
done